src/client/components/immutable-list/immutable-list.tsx

```diff
@@ -128,17 +130,18 @@ export class ImmutableList<T> extends React.Component<ImmutableListProps<T>, Imm
   }
 
   render() {
-    const { items, getRows, label } = this.props;
+    const { items, getRows, label, toggleShowSuggestions } = this.props;
     const { editedIndex, pendingAddItem } = this.state;
 
     if (!items) return null;
-
+    let actions = [<button key='add' onClick={this.addItem.bind(this)}>Add item</button>];
+    if (toggleShowSuggestions) {
+      actions = [<button key='suggestions' onClick={toggleShowSuggestions.bind(this)}>Suggestions</button>].concat(actions);
```

Comment:
no need to bind in `toggleShowSuggestions.bind(this)` since it is coming from outside